README.md

```diff
@@ -2,3 +2,39 @@ BigQuery ETL
 ===
 
 Bigquery UDFs and SQL queries for building derived datasets.
+
+Recommended practices
+===
+
+- Should name sql files like `sql/destination_table_with_version.sql` e.g.
+  `sql/clients_daily_v6.sql`
+- Should not specify a project or dataset in table names to simplify testing
+- Should use incremental queries
+- Should filter input tables on partition and clustering columns
+- Should use UDF language `SQL` over `js` for performance
+- Should use UDFs for reusability
+- Should use query parameters over jinja templating
```

Comment:
Are you mentioning jinja templating because that's something that Airflow provides?

Reply:
Yes, and because this is going to airflow i intend to make any tooling also use jinja templating as well.